Library/Formula/fpp.rb

```diff
@@ -0,0 +1,20 @@
+class Fpp < Formula
+  homepage "https://facebook.github.io/PathPicker/"
+  url "https://facebook.github.io/PathPicker/dist/fpp.0.5.0.tar.gz"
+  sha256 "634cbd2f501639186561418bf2036529d3995676836378b5865e7130d34733e8"
+  head "https://github.com/facebook/pathpicker.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
If the script works on Python 2.6 you don't need this. If the script only works on Python 2.7, make sure the script's shebang uses `/usr/bin/env python`.